flink-java/src/main/java/org/apache/flink/api/java/typeutils/GenericTypeInfo.java

```diff
@@ -70,9 +69,6 @@ public boolean isKeyType() {
 
 	@Override
 	public TypeSerializer<T> createSerializer(ExecutionConfig config) {
-		if(config.serializeGenericTypesWithAvro()) {
```

Comment:
Generic types cannot be serialized with Avro any more? Or all all Avro types POJOs anyways?